.github/workflows/publish.yml

```diff
@@ -162,3 +162,10 @@ jobs:
       - name: Push manifest
         run: |
           make -e V=1 LINUXKIT_PKG_TARGET=manifest pkgs
+
+  trigger_assets:
+    if: ${{ (startsWith(github.ref, 'refs/tags/')) && (github.event.repository.full_name == 'lf-edge/eve') }}
+    needs: [manifest, verification, eve]
+    uses: lf-edge/eve/.github/workflows/assets.yml
+    with:
+      tag_ref: ${{ github.ref }}
```

Comment:
@OhmSpectator I had made changes in the assets.yml file so that it can be triggered by publish.yml by passing the tag. I need to make changes in the assets.yml so that it accepts the input tag and then does publishes the artefacts.

Reply:
So, should we update the assets.yml as well in this PR?.. If this version of publish.yml expects the new version of assets.yml...

Reply:
@OhmSpectator It's done.